src/sidebar/components/test/annotation-thread-test.js

```diff
@@ -27,10 +29,7 @@ describe('annotationThread', function () {
     angular.module('app', [])
       .component('annotationThread', annotationThread)
       .component('moderationBanner', {
-        bindings: {
-          annotationId: '<',
-          isReply: '<',
-        },
+        bindings: moderationBanner.bindings,
```

Comment:
This change ensures that the interface of the `<moderation-banner>` component (ie. its inputs) assumed by this test matches the actual interface.